### PR TITLE
chore(documentation-v7): add card stories

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/documentation-v7/src/stories/components/card/card.demo.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+type Story = StoryObj;
+
+export const BackgroundImage: Story = {
+  ...Default,
+  parameters: {
+    controls: {
+      include: [],
+    },
+  },
```
Comment (imagoiq) [inline packages/documentation-v7/src/stories/components/card/card.demo.stories.tsx:17]: Is it necessary?

File: packages/documentation-v7/src/stories/components/card/card.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+<Canvas of={CardStories.Default}/>
+<div className="hide-col-default">
+  <Controls of={CardStories.Default} />
+</div>
+
+## SCSS Imports
+
+To import all styles from the design-system-styles package:
+
+```css dark
```
Comment (imagoiq) [inline packages/documentation-v7/src/stories/components/card/card.docs.mdx:29]:
```suggestion
```scss
```

File: packages/documentation-v7/src/stories/components/card/card.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+@use '@swisspost/design-system-styles/index.scss';
+```
+
+To import only the styles required for this component:
+
+```css dark
```
Comment (imagoiq) [inline packages/documentation-v7/src/stories/components/card/card.docs.mdx:35]:
```suggestion
```scss
```

**Review by imagoiq — APPROVED** [review]

File: packages/documentation-v7/src/stories/components/card/card.demo.stories.tsx (hunk 1 above)
Comment (alizedebray) [reply packages/documentation-v7/src/stories/components/card/card.demo.stories.tsx:17]: This config removes all controls, it is easier then using `exclude` and listing everything.
We can remove it when going for "docs only".

**Pull request merged**